### DISCUSSION TITLE
Durable data doesn't follow the flow when using the bottom connector

### DISCUSSION THREAD

**Discussion by WilliamWCYoung** [postmanlabs/postman-flows#316] [Defect]
![Screenshot 2022-05-12 at 13 43 59](https://user-images.githubusercontent.com/13030427/168077415-55436c41-858a-4ad0-b4c2-82fa634e1563.png)

When using the top line, I can access the durable data. When using the bottom line, it doesn't. For clarity, I want to use the bottom line as I'm not explicitly passing data from the response of the first block but want to use a durable variable. Is there any reason for this?

**Comment by shrey-bhagat** [comment]
hey @WilliamWCYoung the top line connects to blocks and hence transfers data from one block to another hence you are able to access the durable. While the dashed line is connected to signal, it helps in controlling the execution of the flow. You can read more about it here: https://learning.postman.com/labs/postman-flows/core-concepts/connections/ https://learning.postman.com/labs/postman-flows/getting-started/working-with-access-tokens/

**Reply by WilliamWCYoung** [reply]
But durables aren't part of the main data being transferred between blocks in the flow? It intuitively feels that the durables are, as described, durable for the rest of the flow - which I interpret as the whole flow even through the dashed line signals? Will read your linked article.

**Reply by WilliamWCYoung** [reply]
Read, thank you. I'm still in two minds about why the signal wouldn't transmit durables as it's still a continuation of the flow and durables exist for the lifetime of the flow.
